src/Error/ExceptionRenderer.php

```diff
@@ -184,7 +184,6 @@ public function render()
                 'format' => 'array',
                 'args' => false
             ]);
-            $viewVars['_serialize'][] = 'trace';
```

Comment:
No, that means we still DO by default, but if they contain un-renderable content that it wont fail hard, but gracefully just dont display the trace in that case (as it would result in a white screen of death).